README.md

```diff
@@ -25,45 +25,45 @@ Please read the [Community Participation Guidelines](https://www.mozilla.org/en-
 * Check out the [project wiki](https://github.com/mozilla-mobile/fenix/wiki) for more information.
   * See [our guide on Writing Custom Lint Rules](https://github.com/mozilla-mobile/shared-docs/blob/master/android/writing_lint_rules.md).
 
-* Localization happens on [Pontoon](https://pontoon.mozilla.org/projects/android-l10n/). Please get in touch with delphine (at) mozilla (dot) com directly for more information.
+* Localization happens on [Pontoon](https://pontoon.mozilla.org/projects/android-l10n/). Please get in touch with delphine (at) Mozilla (dot) com directly for more information.
```

Comment:
There was nothing wrong. Email addresses are usually written in lower case.

Reply:
okay, reverted changes.